woocommerce/woocommerce
# Collapse the short description module by default

### All Submissions:






### Changes proposed in this Pull Request:



Closes #33537.

### How to test the changes in this Pull Request:

1. Visit the `Add new` product page
2. When hover the ? icon next to `Add media` button a tooltip should show like below:
<img width="795" alt="image" src="https://user-images.githubusercontent.com/13334210/189414270-639c7664-fb45-4e9b-a380-1cca57753b55.png">
3. When hover the ? icon next to panel with title `Product short description` a tooltip should show like below: 
<img width="795" alt="image" src="https://user-images.githubusercontent.com/13334210/189414880-0913eb15-acd6-4258-8d8e-f763a3c54936.png">
4. Every time the page is visited or refreshed the `Product short description` panel should remain collapsed by default. 
<img width="795" alt="image" src="https://user-images.githubusercontent.com/13334210/189415513-7724df78-b422-4eee-95cd-1c9653951087.png">

### Other information:




### FOR PR REVIEWER ONLY:
